index.html

```diff
@@ -216,9 +223,9 @@ <h2>Terminology</h2>
 [[HR-TIME-2]]. For example, the <a data-cite=
 "NAVIGATION-TIMING-2#dom-PerformanceNavigationTiming-startTime">start
 of navigation of the document</a> occurs at time 0.</p>
-<p>The term <dfn>current time</dfn> refers to the number of
-milliseconds since the start of navigation of the document until
-the current moment in time.</p>
+<p>The term <dfn>current time</dfn> refers to the result of running 
+<a data-cite="HR-TIME-ED#dfn-current-high-resolution-time">current high 
+resolution time</a> where |current global| is the {{Window}} object.</p>
```

Comment:
Maybe you can add an inline issue (`<p class=issue>` IIRC) to align this to the published spec, once we've actually published it

Reply:
Probably not needed as this will be on L3, right ?

Reply:
indeed!